### DISCUSSION TITLE
Or Condition for conditional cards

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#20478] [Dashboard]
It would be great, if a conditional card could have an OR conditon - so either a OR b could be true to display the content

For example, this would be usefull for my plant cards / Dashboard:

Example:
Show the plant when:
Temperature is too low OR too high
OR
Moisture is too low OR too high
OR
conductivity is too low OR too high
OR
battery is empty

**Comment by karwosts** [comment]
Already exists: https://www.home-assistant.io/dashboards/conditional/#or
